source/FreeRTOS_IP_Timers.c

```diff
@@ -604,3 +604,13 @@ void vIPSetARPResolutionTimerEnableState( BaseType_t xEnableState )
 
 #endif /* ipconfigDNS_USE_CALLBACKS == 1 */
 /*-----------------------------------------------------------*/
+
+/**
+ * @brief Mark that at least one interface is down so that the 'xNetworkTimer' is checked.
+ *        Whenever the timer expires, all interfaces that are down will get a new NetworkDown
+ *        event.
+ */
+void vSetNotAllNetworksUp( void )
```

Comment:
Can you please update the function name to be more clearer.

Reply:
any suggestions?

Reply:
We can make it generic, pass the argument. vSetAllNetworksUp with input parameter as xIsAllNetworksUp

Reply:
how about now?

Reply:
Yeah seems better. You might have to add Unit Test for the new function you have added for code coverage.

Reply:
I'll fix the existing unit tests and add more tests to cover new functions once the solution is accepted.